modin/backends/pandas/parsers.py

```diff
@@ -230,12 +213,16 @@ def parse(fname, **kwargs):
         ]
 
 
-@Appender(
-    _doc_pandas_parser_class.format(data_type="multiple CSV files simultaneously")
-)
+@doc(_doc_pandas_parser_class, data_type="multiple CSV files simultaneously")
 class PandasCSVGlobParser(PandasCSVParser):
     @staticmethod
-    @Appender(_doc_parse_func.format(parameters=_doc_parse_parameters_csv_glob))
+    @doc(
+        _doc_parse_func,
+        parameters="chunks : list\n"
+        "    List, where each element of the list is a list of tuples. The inner lists\n"
+        "    of tuples contains the data file name of the chunk, chunk start offset, and \n"
+        "    chunk end offsets for its corresponding file.",
```

Comment:
```suggestion
        parameters="""chunks : list
    List, where each element of the list is a list of tuples. The inner lists
    of tuples contains the data file name of the chunk, chunk start offset, and
    chunk end offsets for its corresponding file.""",
```
This is what multiline strings were invented for 😉